### ISSUE TITLE
Discord web client UI

### ISSUE THREAD

**Issue by azureus89** [224-1/xtnt-micro#38]
**20 credits:**
clj template
componentised hiccup
data driven

**+10 credits:**
responsive

**Deadline: 23/9/19**

**Notes**
ask me if you want me to specify the data model and you dont want to attempt it yourself

**Comment by ParikshithMohan** [comment]
data model?

**Comment by azureus89** [comment]
yes check the trelloui componentised version, components are drive by state atom ... that is the core part of a reactive application

**Comment by ParikshithMohan** [comment]
got it.

**Comment by ParikshithMohan** [comment]
Progres...

![image](https://user-images.githubusercontent.com/24820582/65392628-6b809d00-dd94-11e9-9b00-c4e1ba5e1c74.png)

**Comment by ParikshithMohan** [comment]
![image](https://user-images.githubusercontent.com/24820582/65407583-a6b8b580-de00-11e9-8255-87c7af1914c5.png)

**Comment by azureus89** [comment]
Pretty good, attention to detail much much better than previous first cuts, but learn to polish it, subtler things missing, try to spot them by comparing.

how componentised is your code? We will be stopping work on this tomorrow and moving to bomb ... so focus on componentisation till then, not UI improvements .. 

you can just reply with the answer to above (spot subtle changes), no need to work on them, will get @kattakavya to take over, once its componentised and continue from there. It's important that both of you are on the same page of understanding of how the system comes together.

She will have some documentation demands of you, so be prepared for that. If your code is messy like before, its only going to create more work for you later, better to address it now methodically.

**Comment by ParikshithMohan** [comment]
**IMPROVEMENTS TO BE MADE:**
-Font used by discord is Whitney. I used Lato since Whitney is a paid font.
-Left-most container `:hr` width not right and xtnt logo `border-left` not perfect.
-right-most container: No green and moon logo to show online or not.
-Did not find perfectly similar icons...for example the smiley
-Color inside chat container slightly off (Need to refer the color hex values I have added in doc called discord-hex.org)
-Font weights not exactly similar in some places
-No customized scrollbar in chat container

**Comment by ParikshithMohan** [comment]
@azureus89  How do I tell if my the level of componentization is enough or not? Not sure of the amount of componentization u want... also don't know if I overdid it.  I have tried to make the code much less messy this time. When should I make a pull request?

**Comment by azureus89** [comment]
@ParikshithMohan whenever you feel its ready ... my review will come only after pull request
you have time till tomorrow

**Comment by azureus89** [comment]
@ParikshithMohan did you try responsive?

**Comment by ParikshithMohan** [comment]
No, I didn't.

**Comment by azureus89** [comment]
@sammed14 take this forward and do responsive ... @ParikshithMohan help sammed understand how the whole thing is organised -- how clojure templating works

**Comment by azureus89** [comment]
@sammed14 any permission issues on this repo, let me know

**Comment by azureus89** [comment]
@sammed14 confirm ready to pick this up from 25th
will detail improvements required in this thread

**Comment by azureus89** [comment]
@sammed14 whenever you are ready

**Comment by azureus89** [comment]
@sammed14 *yawn*

**Comment by azureus89** [comment]
@kattakavya use this issue ... make ui rseponsive
